app/tests/integration/nimbus/conftest.py

```diff
@@ -121,9 +121,10 @@ def experiment_url(base_url, default_data, slugify):
 def default_data(request):
     application = request.param
     feature_config = APPLICATION_FEATURES[application]
+    public_name = f"{request.node.name}-{str(uuid.uuid4())[:4]}"
 
     return BaseExperimentDataClass(
-        public_name=f"{request.node.name}-{str(uuid.uuid4())[:4]}",
+        public_name=public_name[:80],
```

Comment:
Oh yeah I was thinking about this and I realized if we do this way then we could chop off the UUID portion altogether, I think we should be trimming `request.node.name[:76]` so we always include the UUID at the end to prevent name collisions on test reruns.

Reply:
doh, good one!